src/ai/backend/manager/models/vfolder.py

```diff
@@ -108,8 +149,8 @@ class VFolderInvitationState(str, enum.Enum):
     sa.Column('invitee', sa.String(length=256), nullable=False),  # email
     sa.Column('state', EnumValueType(VFolderInvitationState),
               default=VFolderInvitationState.PENDING),
-    sa.Column('created_at', sa.DateTime(timezone=True),
-              server_default=sa.func.now()),
+    sa.Column('created_at', sa.DateTime(timezone=True), server_default=sa.func.now()),
+    sa.Column('modified_at', sa.DateTime(timezone=True), nullable=True),
```

Comment:
It seems that there is no place to update this field.

Reply:
Make `modified_at` updated whenever mutation performed. The value will also be returned when querying virtual folders.